### PR TITLE
submit hw1

### PR DESCRIPTION
**Pull request by yiyang-leo-S**
hw1 submission for review

### CODE REVIEW

File: Homeworks/HW1/yiyang_song/game_of_life.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+def evolve(initial_state):
+    import copy
```
Comment (pusukuri) [inline Homeworks/HW1/yiyang_song/game_of_life.py:2]: Please move all the import statements to the top of the file.

**Review by pusukuri — APPROVED** [review]
Beautiful code. Great job.

File: Homeworks/HW1/yiyang_song/game_of_life.py (hunk 1 above)
Comment (yiyang-leo-S) [reply Homeworks/HW1/yiyang_song/game_of_life.py:2]: Thanks for the suggestion, do I need to resubmit?

Comment (pusukuri) [reply Homeworks/HW1/yiyang_song/game_of_life.py:2]: No need. However, let us improve coding in the future assignments :-)

**Pull request merged**